.github/ISSUE_TEMPLATES/new-goal.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: "🧭 New team goal"
+about: Creating a new team goal
+labels: goal
+---
+
+# Background
+
+<A one to two sentence description of the goal and other background as-needed. Try to keep it short!>
+
+# Expected timeline
+
+<How long do you imagine this goal should be active for? This will determine the cadence with which we should check up on it to see how things are going.>
+
+# How will we know when this goal is complete?
+
+<This can be an abstract end-point, a date, a checklist of issues, etc. Something that will let us know when to close the issue.>
+
+# Related issues and links
```

Comment:
When writing https://github.com/2i2c-org/pilot-hubs/issues/261, I realized that it would have been more practical to have the last two sections (`How will we know when the goal is complete` and `Related issues and links`) should have been just one. Link an issue (if available) to each item in the checklist.